### PR TITLE
BREAKING-CHANGE: Renamed variables-scss to tokens-exporter and add assets-exporter #DS-1483

### PR DESCRIPTION
**Pull request by curdaj**

### CODE REVIEW

File: .commitlintrc.mjs [hunk 1]
```diff
@@ -24,7 +24,8 @@ export default {
         'exporter-js',
         'exporter-scss',
         'exporter-svg',
-        'exporter-variables-scss',
+        'spirit-assets-exporter',
```
Comment (literat) [inline .commitlintrc.mjs:27]: I prefer `exporter-tokens` and `exporter-assets`. I think we do not need to use the `spirit` prefix here. So the names remained short and sorted based on context (`exporter-`).

File: .eslintrc.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ module.exports = {
     'packages/codemods',
     'exporters/scss',
     'exporters/js',
-    'exporters/variables-scss',
+    'exporters/spirit-tokens-exporter',
```
Comment (literat) [inline .eslintrc.js:31]:
```suggestion
    'exporters/tokens',
```

File: .prettierignore [hunk 3]
```diff
@@ -78,4 +78,4 @@ CODEOWNERS
 **/__fixtures__/unformatted*
 
 # variable-scss exporter generated cjs
-exporters/variables-scss/generated/**/*.cjs
+exporters/spirit-tokens-exporter/generated/**/*.cjs
```
Comment (literat) [inline .prettierignore:81]:
```suggestion
exporters/tokens/generated/**/*.cjs
```

**Comment by literat** [comment]
I suggest use only `tokens` for directory name. So the structure will be `exporters/tokens`. Clean and short. We do not use the `spirit-` prefix for any package directory, so here we should apply the same rule. :-)

**Comment by pavelklibani** [comment]
rebased with main (fixed supernova bug - updated versions etc.)

**Review by pavelklibani — APPROVED** [review]

File: exporters/assets/exporter.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
+{
+  "id": "eu.lmc.spirit-assets-exporter",
```
Comment (literat) [inline exporters/assets/exporter.json:2]:
```suggestion
  "id": "alma.career.spirit-assets-exporter",
```

File: exporters/assets/exporter.json [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  "name": "Spirit Assets Exporter",
+  "description": "Spirit Assets Exporter",
```
Comment (literat) [inline exporters/assets/exporter.json:4]: Maybe a better description?

File: exporters/assets/exporter.json [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+  "author": "Spirit Team",
```
Comment (literat) [inline exporters/assets/exporter.json:5]:
```suggestion
  "author": "Spirit Design System Team <team_design_system@almacareer.com>",
```

File: exporters/tokens/exporter.json [hunk 7]
```diff
@@ -1,7 +1,7 @@
 {
-  "id": "almacareer.exporter-spirit-variables-scss",
-  "name": "Spirit Varibles SCSS Exporter",
-  "description": "Spirit SCSS Exporter",
+  "id": "almacareer.spirit-tokens-exporter",
```
Comment (literat) [inline exporters/tokens/exporter.json:2]:
```suggestion
  "id": "alma.career.spirit-tokens-exporter",
```

`alma.career` or `almacareer` 🤷‍♂️ but pick for consistency in both `exporter.json`.

File: exporters/tokens/exporter.json [hunk 8]
```diff
@@ -1,7 +1,7 @@
...
+  "name": "Spirit Tokens Exporter",
+  "description": "Spirit Tokens Exporter",
```
Comment (literat) [inline exporters/tokens/exporter.json:4]: Maybe a better description?

**Review by literat — APPROVED** [review]
Great Job. It is looking good. I have made a few nice-to-have suggestions.

Also, it comes to mind that it might be appropriate to deprecate the `svg`, `js`, and `scss` exporters. I mean like write some sentence into the exporters readmes: `This exporter is deprecated. Use ... instead.` And link to the new exporter code.
I know that is under our control, but for history's sake :-) Thanks.

**Comment by crishpeen** [comment]
> Great Job. It is looking good. I have made a few nice-to-have suggestions.
> 
> Also, it comes to mind that it might be appropriate to deprecate the `svg`, `js`, and `scss` exporters. I mean like write some sentence into the exporters readmes: `This exporter is deprecated. Use ... instead.` And link to the new exporter code. I know that is under our control, but for history's sake :-) Thanks.

I don't think so we want to deprecate these exporters in `main` yet. We do not offer an alternative and teams need them.

We can do this when we merge v3 (or we can deprecate it in the integration branch).

File: exporters/assets/exporter.json [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+  "organization": "Alma Career",
+  "source_dir": "src",
+  "version": "2.1",
```
Comment (crishpeen) [inline exporters/assets/exporter.json:8]: maybe we should start with some major version. Either 1 or 3.

File: exporters/assets/src/asset_path.pr [hunk 10]
```diff
@@ -0,0 +1,9 @@
+{[ let extension = asset.format /]}
+
+{* Generate name as [originalName][-numberOfDuplicates][extension] *}
+{[ let duplicateExtension = (asset.previouslyDuplicatedNames > 0 ? ("-" + asset.previouslyDuplicatedNames) : "") /]}
+{[ let name = (asset.originalName.lowercased().replacing(" ", "-") + duplicateExtension +  "." + asset.format) /]}
+   
+{[ if asset.group.name.equals("Icons") ]}
+    {{ name }}
+{[/]}
```
Comment (crishpeen) [inline exporters/assets/src/asset_path.pr:9]: an empty line at the end of the file please

File: exporters/tokens/package.json [hunk 11]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@almacareer/spirit-exporters-variables-scss",
-  "version": "1.2.0",
-  "description": "Spirit SCSS Exporter for Supernova",
+  "name": "@almacareer/spirit-tokens-exporter",
+  "version": "1.1.0",
+  "description": "Spirit Tokens Exporter for Supernova",
   "author": "Spirit Team",
```
Comment (crishpeen) [inline exporters/tokens/package.json:5]: same author as in assets?

**Review by crishpeen — COMMENTED** [review]
Please update the commit message scope in the first commit.

**Pull request merged**